meta-genivi-dev/recipes-dev-hmi/genivi-dev-platform-hmi/notepad-app.bb

```diff
@@ -15,6 +15,7 @@ inherit qmake5
 
 SRC_URI_append ="\
     file://notepad-app.desktop \
+    file://0001-Notepad.pro-Remove-usr-includes-from-include-path.patch \
```

Comment:
Because I do not know if this patch would work for other solutions than the GDP Yocto build.